analyzer_test.go

```diff
@@ -66,33 +65,35 @@ var _ = Describe("Analyzer", func() {
 					println("package has two files!")
 				}`)
 			pkg.Build()
-			err := analyzer.Process(buildTags, pkg.Path)
+			err := analyzer.Process(buildTags, pkg.Path+"/foo.go", pkg.Path+"/bar.go")
 			Expect(err).ShouldNot(HaveOccurred())
 			_, metrics, _ := analyzer.Report()
 			Expect(metrics.NumFiles).To(Equal(2))
 		})
 
-		It("should be able to analyze multiple Go packages", func() {
-			analyzer.LoadRules(rules.Generate().Builders())
-			pkg1 := testutils.NewTestPackage()
-			pkg2 := testutils.NewTestPackage()
-			defer pkg1.Close()
-			defer pkg2.Close()
-			pkg1.AddFile("foo.go", `
-				package main
-				func main(){
-				}`)
-			pkg2.AddFile("bar.go", `
-				package main
-				func bar(){
-				}`)
-			pkg1.Build()
-			pkg2.Build()
-			err := analyzer.Process(buildTags, pkg1.Path, pkg2.Path)
-			Expect(err).ShouldNot(HaveOccurred())
-			_, metrics, _ := analyzer.Report()
-			Expect(metrics.NumFiles).To(Equal(2))
-		})
+		/*
+			It("should be able to analyze multiple Go packages", func() {
+				analyzer.LoadRules(rules.Generate().Builders())
```

Comment:
Same comment here regarding `Skip()`

Reply:
Shouldn't this test pass? In a Go module are most likely multiple packages.